docker-compose.test.yml

```
@@ -22,16 +30,21 @@
 
   apollo:
     build:
       context: .
       dockerfile: Apollo.Dockerfile
     restart: unless-stopped
-    volumes:
-      - .:/app
     ports:
       - '4000:4000'
+    healthcheck:
+        <<: *default-healthcheck
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 8

(indentation)